### DISCUSSION TITLE
configuration error

### DISCUSSION THREAD

**Discussion by doudou49000** [jagheterfredrik/wallbox-mqtt-bridge#57] [Q&A] [answered]
hello

First of all, thank you for posting, I think your work is great, I followed the tutorials and I succeeded even without any knowledge.

I made a mistake when entering the IP "192.168.10.165:1883" instead of "192.168.10.165".
I am unable to re-edit the configuration.
I'm not good at computers, to say this is the first time I've touched MQTT and even more so ROOT.
is there a file to delete in the terminal? or an action?
Thanks for your help.

**Comment by tronikos** [comment, marked as answer]
You can edit `/home/root/mqtt-bridge/bridge.ini` or delete it and re-install.

**Comment by doudou49000** [comment]
THANKS 
I preferred to delete the file because I don't know how to open them lol.
I installed it again and everything went up automatically in MQTT. I will be able to continue programming.
Thanks again for the work.
